backend/app/.env.example

```diff
@@ -18,3 +18,6 @@ APS_REDIS_DATABASE=1
 # Token
 TOKEN_SECRET_KEY='1VkVF75nsNABBjK_7-qz7GtzNy3AMvktc9TCPwKczCk'
 TOKEN_WHITE_LIST=[1]
+# Test
+TEST_USERNAME='test'
+TEST_USER_PASSWORD='123456'
```

Comment:
Will test-exclusive data be included only in the test?

Reply:
It can also be used to initialize data

Reply:
Sounds good.

But I don't think the test data should be in the environment, just in the config, as this is only for testing purposes